docs/cpu_templates/cpuid-normalization.md

```diff
@@ -0,0 +1,77 @@
+# CPUID normalization
+
+Firecracker makes certain modifications to the guest's CPUID regardless of
+whether a CPU template is used. This is referred to as `CPUID normalization`.
+If a CPU template is used the normalization is performed _after_
+the CPU template is applied. That means that if the CPU template configures
+CPUID bits used in the normalization process, they will be overwritten.
+
```

Comment:
How about adding references for when users want to look for more details (e.g., Intel SDM and AMD APM)?

Reply:
Added.